docs/docs/getting-started/05-examples/03-mock-hardware-example.md

```diff
@@ -0,0 +1,206 @@
+---
+sidebar_position: 3
+---
+
+# An application with hardware
+
+## Package requirements
+
+This example requires the Universal Robots collection which includes example URDF content.
+
+```toml title="aica-package.toml"
+#syntax=ghcr.io/aica-technology/app-builder:v1
+
+[packages]
+"@aica/base" = "v2.1.0"
+
+"@aica/collections/ur-collection" = "v3.0.4"
```

Comment:
Nitpick: Maybe also add an example of how to put self packaged files? For example, I import my changed abb gofa hardware component with image tag _tl_ as:

`"@aica/collections/abb-gofa-collection" = "docker-image://ghcr.io/aica-technology/collections_abb-gofa_collection:tl"`

Reply:
I'm not sure if it's useful in this example, but perhaps there were changes made to the ur-collection? Or maybe this will go better somewhere else?

Reply:
that's a more advanced use-case and is better suited to the programming reference section. In some ways it's already covered here:
https://docs.aica.tech/docs/reference/custom-components/aica-package-toml

Reply:
Agree with the suggestion and the answer. This is not the place for it.